src/Stratis.Bitcoin.Features.SmartContracts/ApiLogDeserializer.cs

```diff
@@ -55,6 +55,9 @@ public dynamic DeserializeLogData(byte[] bytes, Type type)
                 {
                     object fieldValue = this.primitiveSerializer.Deserialize(fieldType, fieldBytes);
 
+                    if (fieldType == typeof(UInt128) || fieldType == typeof(UInt256))
```

Comment:
You already fixed json deserialization issue by ToStringJsonConverter class so this code is not necessary.

Reply:
This was necessary for logs to return correctly, without it, return values from a `local-call` would be correct but logs would still return `{}`.